.github/workflows/ci.yml

```diff
@@ -268,7 +268,9 @@ jobs:
       - filter-changes
       - dockerpush
     timeout-minutes: 90
-    concurrency: ${{ github.workflow }}-${{ github.ref }}
+    concurrency:
+      group: ${{ github.base_ref == 'staging' && format('{0}-staging', github.workflow) || format('{0}-{1}', github.workflow, github.ref) }}
+      cancel-in-progress: ${{ github.base_ref == 'staging' && false || true }}
```

Comment:
What is going on here: 
```
&& false || true
```

Reply:
That's a github workaround for NOT having ternary operator... 
it's basically `condition && if-true || if-false`.